flux.json

```diff
@@ -0,0 +1,21 @@
+{
+  "homepage": "https://justgetflux.com/",
+  "description": "Makes the color of your computer's display adapt to the time of day, warm at night and like sunlight during the day",
+  "version": "4.66",
+  "url": "https://justgetflux.com/flux-setup.exe",
+  "hash": "79fc802d9225d98ee15a288393bb2d4708575315b9edad33dbfcf29d1af578b1",
+  "bin": "flux.exe",
+  "installer": {
+    "script": "cmd /c start /WAIT $dir\\flux-setup.exe /S /D=$dir"
```

Comment:
Why do you start the setup via `cmd` instead of using the `file` property?

Reply:
flux auto-starts immediately after installing, and so causes scoop installation to hang. not sure if there is an arg to pass to keep from auto-starting. Also not sure if there is a better way to handle this from command line.

Reply:
you're meaning using something like 

```
  "installer": {
    "file": "flux-setup.exe",
    "args": ""
```

Reply:
Good point with the blocking installer.
But could you add quotes around the setup exe? This prevents issues with whitespaces in `$dir` (`/D=$dir` doesn't need quotes)
`cmd /c start /WAIT \"$dir\\flux-setup.exe\" /S /D=$dir`

Reply:
done! thanks